devlib/utils/ssh.py

```diff
@@ -175,7 +180,8 @@ def pull(self, source, dest, timeout=30):
         source = '{}@{}:{}'.format(self.username, self.host, source)
         return self._scp(source, dest, timeout)
 
-    def execute(self, command, timeout=None, check_exit_code=True, as_root=False, strip_colors=True):
+    def execute(self, command, timeout=None, check_exit_code=True,
+                as_root=False, strip_colors=True, wrap_busybox=False):
```

Comment:
The `wrap_busybox` and `strip_colors` parameter order is different to the `execute` method for gem5connection.